oem/ibm/libpldmresponder/oem_ibm_handler.hpp

```diff
@@ -595,6 +632,9 @@ class Handler : public oem_platform::Handler
 
     int setEventReceiverCnt = 0;
 
+    /** @brief Real SAI sensor id*/
+    uint16_t realSAISensorId;
```

Comment:
not needed if function findSensorId moved?

Reply:
Cannot move as repo is passed from void pldm::responder::oem_ibm_platform::Handler::buildOEMPDR() and not present under class Handler